Assets/MRTK/Providers/LeapMotion/Editor/LeapMotionConfigurationChecker.cs

```diff
@@ -232,6 +240,11 @@ private static void AddAndUpdateAsmDefs()
                 if (currentLeapCoreAssetsVersion == "4.5.1")
                 {
                     leapAsmDef.AddReference("LeapMotion.LeapCSharp");
+
+                    // If the unity modules version is 4.6.0 or 4.7.0 then add SpatialTracking as a reference
+#if UNITY_2019_3_OR_NEWER
+                    leapAsmDef.AddReference("UnityEngine.SpatialTracking");
```

Comment:
Should this check against `currentLeapCoreAssetsVersion` for 4.6.0/4.7.0? (based on the comment)

Reply:
It should be checking against the version number but in the PR description I mentioned that the Version.txt has remained 4.5.1 for the 4.6.0, 4.7.0 and 4.7.1 release. 

4.6.0 release branch : https://github.com/leapmotion/UnityModules/blob/rc-4.6.0/Assets/Plugins/LeapMotion/Core/Version.txt

4.7.0 release branch : https://github.com/leapmotion/UnityModules/blob/release-4.7.0/Assets/Plugins/LeapMotion/Core/Version.txt

4.7.1 release branch: https://github.com/leapmotion/UnityModules/blob/4.7.1-partners/Assets/Plugins/LeapMotion/Core/Version.txt

It would be great if Leap could change this.

Reply:
Updated the version inference without the Version.txt in the latest commit.